src/main/resources/templates/smallfull/balanceSheet.html

```diff
@@ -61,7 +61,37 @@ <h2 class="govuk-heading-s">
                         </div>
                     </div>
 
-                    <br>
+                    <!--Fixed Assets-->
+
+                    <h2 class="govuk-heading-m">Fixed assets</h2>
+
+                    <div class="govuk-grid-row">
```

Comment:
Should there be an error row above this?

Reply:
Was going to add the validation in my next PR so will add the error row then